_posts/2015-03-15-git2go.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Git2Go"
+category: app
+icon: "/assets/Git2Go.svg"
+app_store_link: "http://git2go.com"
+---
+
+## Is it released yet? I am waiting for it.
+
+@tapwork, [Heap Inspector](https://github.com/tapwork/HeapInspector-for-iOS) and iOS Developer
```

Comment:
@tapwork are you ok with your mention?
![tapwork_quote](https://cloud.githubusercontent.com/assets/759730/7378043/eb68b816-edea-11e4-8905-31ff36fbeae2.png)